src/reducer.js

```diff
@@ -200,17 +200,22 @@ export default function undoable (reducer, rawConfig = {}) {
           return config.history
         }
 
+        const wasFiltered = (typeof config.filter === 'function' && !config.filter(action, res, history))
+
         if (history.present === res) {
           // Don't handle this action. Do not call debug.end here,
           // because this action should not produce side effects to the console
-          return history
+          return {
+            ...history,
+            wasFiltered
```

Comment:
Adding this would update the `wasFiltered` value on every single action that passes through the root reducer, including actions not meant for the `undoable` reducer. I don't think we want to run the filter on every single action.